### PR TITLE
Organizer can manage tracks

### PR DESCRIPTION
**Pull request by gitcow**
Extended track crud to include new fields and edit capability.

### CODE REVIEW

**Comment by gitcow** [comment]
Basic crud's in place. Couple issues to address:
- no breadcrumb/path back to main event page. Necessary now? Same with session type crud.
- no cancel button on form (ditto session type). Again, not a big deal that it's missing, but easy to fix.
- at least one broken test. Should I update existing tests or wait until the overall flow is worked out?
- had second thoughts about doing basic crud as opposed to building on the ajax modal approach that was already in place. I like ajax.
- moved track and session type buttons to the top-right sub-nav area. OK for now?

File: app/controllers/organizer/session_types_controller.rb [hunk 1]
```diff
@@ -48,7 +46,12 @@ def destroy
   private
 
   def set_session_type
-    @session_type = @event.session_types.find(params[:id])
+    if params[:id]
```
Comment (mghaught) [inline app/controllers/organizer/session_types_controller.rb]: This is a curious change.  What's the benefit of bring new and find logic together?

**Comment by mghaught** [comment]
This looks good overall and we'll be addressing the page flow shortly in another card since the event edit/management page is a mess.  We will want to put in a cancel button since that's traditional but that can be caught on a follow-up card that will deal with styles and the general breakout of the edit event functionality.

File: app/controllers/organizer/session_types_controller.rb (hunk 1 above)
Comment (gitcow) [inline app/controllers/organizer/session_types_controller.rb]: Oh just saw this. I was thinking of it as a way to flatten out the controller expectation of @session_type being set for all actions that use it. I figured it was either this or write a separate before_action for the 'new' case. Not a terribly important thing I suppose.

**Pull request merged**